tests/__init__.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from babel._compat import BytesIO
```

Comment:
`babel==2.10.1` dropped the `babel._compat` module. According to the [babel docs](https://github.com/python-babel/babel/pull/808/files#diff-f5b6d24b6b04bab5e6af9b851f476c8f61261df95b0258b9d789a71170574576R261) we can use `io` for this functionality instead.